tools/scripts-config-eslint/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@talend/scripts-config-eslint",
+  "name": "@talend/eslint-config",
```

Comment:
this change has not been done in other script-config-* folders

Reply:
this change has been done to comply with eslint implicit research ('@talend') so not needed for others.